addons/mail/models/mail_thread.py

```diff
@@ -3970,6 +3970,24 @@ def _notify_by_web_push_prepare_payload(self, message, msg_vals=False):
         else:
             icon = '/web/static/img/odoo-icon-192x192.png'
 
+        if not body and not message.body and message.attachment_ids:
+            total_attachments = len(message.attachment_ids)
+            first_attachment = message.attachment_ids[0]
+            first_attachment_label = _('Voice message') if first_attachment.voice_ids else first_attachment.name
+
+            if total_attachments == 1:
+                body = first_attachment_label
+            else:
+                body = self.env._(
+                    "%(attachment_name)s and %(count)d other attachment.",
+                    attachment_name=first_attachment_label,
+                    count=total_attachments - 1,
+                ) if total_attachments == 2 else self.env._(
+                    "%(attachment_name)s and %(count)d other attachments.",
+                    attachment_name=first_attachment_label,
+                    count=total_attachments - 1,
+                )
```

Comment:
The case for 2 and more are currently having the exact same text?
It's supposed to be "attachment 1" and "attachment 2" for the "2" case rather than displaying one name and then count.